source/layouts/layout.erb

```diff
@@ -8,7 +8,7 @@
     <%= stylesheet_link_tag 'index.bundle' %>
     <%= javascript_include_tag 'head.bundle' %>
   </head>
-  <body class="<%= page_classes %> grid-container fluid">
+  <body class="<%= page_classes %> grid-container">
```

Comment:
Why are we removing the fluid resizing here?

Reply:
Because Ahliah had some valid criticism with the text expanding too far on larger screens -- I agreed with it, and Foundation allows removing the "fluid" keyword to limit the width of the site to 1200px. My perception is that it looks better on larger screens and doesn't affect usability, but you're more knowledgeable about that than I am.

Reply:
Ooh, that makes complete sense. I applied the `fluid` class thinking along the lines of an older version of the bootstrap framework, where `fluid` (or similar) made it responsive whereas it was set to a hard width regardless of viewport otherwise.

If this keeps it responsive but sets a max width of 1200px, I'd agree it'd be better to remove it. 👍